lab_6_pipeline/pipeline.py

```diff
@@ -88,6 +144,16 @@ def _bootstrap(self) -> AbstractCoNLLUAnalyzer:
         Returns:
             AbstractCoNLLUAnalyzer: Analyzer instance
         """
+        model = spacy_udpipe.load_from_path(
+            lang="ru",
+            path=str(model_path)
```

Comment:
What is `model_path`? Where is it defined?

Reply:
i'll put correct path of course